scripts/mesh-point/mesh-point

```diff
@@ -18,6 +18,12 @@ function isMeshable {
 
 set -e
 
+mesh_name=$(confget -f /etc/mesh.conf -s general "mesh-name")
+frequency=$(confget -f /etc/mesh.conf -s general "frequency")
+if [ -z "$frequency" ]; then
+  frequency=2412
+fi
+
 # Set wireless regulatory domain
 sudo iw reg set CA
```

Comment:
This should be a question we ask the user, and then something set in `/etc/mesh.conf`.

Reply:
No because this is the service so there is no UI component.

Default frequency if one is not set is 2412

Reply:
No but I mean let's ask the user in `install2`, set it in `/etc/mesh.conf`, and then read from that file here.

Reply:
that can be a ui thing.